src/pages/teams.astro

```diff
@@ -5,325 +5,723 @@ import Layout from "../layouts/Layout.astro";
+                    <h3 class="text-lg md:text-3xl font-semibold ml-4">
+                      Encrypted secrets
+                    </h3>
+                  </div>
+                  <p
+                    class="text-md md:text-lg font-light max-w-sm ml-[52px] mt-2 text-gray-700"
+                  >
+                    Save API and database credentials with Livebook Secrets,
+                    which are encrypted with a key to which only you have
+                    access.
+                  </p>
                 </div>
-                <span class="lg:w-[70%] text-gray-200">
-                  If you have any questions about Livebook Teams, we're here to
-                  help.
-                </span>
-                <a class="button button-gray">
-                  <span>Send an email</span>
-                </a>
               </div>
             </div>
           </div>
         </div>
-      </section>
-    )
-  }
-</Layout>
+      </div>
+    </div>
+  </section>
+  <section
+    id="cta"
+    class="bg-gray-100 pt-[410px] sm:pt-96 md:pt-72 pb-20 sm:pb-32 px-4 lg:px-0"
+  >
+    <div class="flex flex-col gap-10 items-center">
+      <h1
+        class="max-w-xl text-3xl/tight sm:text-4xl/tight md:text-5xl/tight font-semibold text-center"
+      >
+        Get started building internal tools in minutes
+      </h1>
+      <a
+        class="button-cta button-pink plausible-event-name=teams+early+access+main+cta+click drop-shadow-[0_15px_25px_rgba(228,76,117,0.5)] hover:drop-shadow-[0_15px_25px_rgba(0,0,0,0.5)]"
+        href="https://docs.google.com/forms/d/e/1FAIpQLScDfvUqT4f_s95dqNGyoXwVMD_Vl059jT6r5MPgXB99XVMCuw/viewform"
+      >
+        <Icon icon="rocket-line" class="text-xl" />
+        <span class="px-2 font-semibold">I want FREE early access</span>
+      </a>
+    </div>
+  </section>
+  <section id="video-lightbox">
+    <Lightbox
+      dataLightbox="1"
+      videoURL="https://www.youtube.com/embed/E27-3MwY7KU?si=YoCaNlDGKQW10WzB"
+    />
+    <Lightbox
+      dataLightbox="2"
+      videoURL="https://www.youtube.com/embed/z5GziuQoUxk?si=jIFd6eSrkYSlJ0ah"
+    />
+  </section>
 
-<script>
-  document.querySelectorAll(".faq-question-btn").forEach((button) => {
-    button.addEventListener("click", () => {
-      const parent = button.parentElement;
-      const answer = parent.querySelector(".faq-answer");
-      const iconDown = parent.querySelector(".faqIconDown");
-      const iconUp = parent.querySelector(".faqIconUp");
+  <script>
+    document.addEventListener("DOMContentLoaded", () => {
+      const openLightboxButtons = document.querySelectorAll(".open-lightbox");
+      const lightboxes = document.querySelectorAll(".lightbox");
+      const closeLightboxButtons = document.querySelectorAll(".close-lightbox");
 
-      answer.classList.toggle("hidden");
-      iconDown.classList.toggle("hidden");
-      iconUp.classList.toggle("hidden");
+      openLightboxButtons.forEach((button) => {
```

Comment:
I would change the naming in general from `Lightbox` to `VideoModal` or similar (since the component specifically handles videos, that makes the code more clear :)).